metamorph-api/src/main/java/org/metafacture/metamorph/api/helpers/AbstractReadOnlyMap.java

```diff
@@ -41,7 +42,7 @@ public final boolean isEmpty() {
     }
 
     @Override
-    public final boolean containsKey(final Object key) {
+    public boolean containsKey(final Object key) {
```

Comment:
This doesn't really have anything to do with the intended change, does it?

Reply:
Yes. It results from https://github.com/metafacture/metafacture-core/pull/416#issuecomment-961938387.
Is it ok for you if this would be a separate commit but stays in this PR? And do you agree that I can force push that here?
(Note for self re commits::  take https://github.com/metafacture/metafacture-core/blob/master/CONTRIBUTING.md#git-conventions serious)

Reply:
> Yes. It results from [#416 (comment)](https://github.com/metafacture/metafacture-core/pull/416#issuecomment-961938387).

That was in the context of "chang[ing] all reading methods of 'AbstractReadOnlyMap' to not be final", which doesn't apply here anymore. But I won't oppose whatever you decide.

Reply:
> And do you agree that I can force push that here?

Always yes from me :)